Makefile

```diff
@@ -64,22 +63,23 @@ go-generate: install-tools
 .PHONY: unit-test
 unit-test: go-generate ## Run unit tests
 	go test $(gotest_extra_flags) -race -v \
-			-coverprofile=coverageunit.out \
-			-coverpkg=$(COVERPKG) \
-			-covermode=atomic -race \
+			-coverpkg=./... \
+			-coverprofile=coverageunit.tmp.out \
+			-covermode=atomic \
 			-count=1 \
+			-timeout=5m \
 			./...
+	@cat coverageunit.tmp.out | grep -v "mocks" > coverageunit.out
+	@rm coverageunit.tmp.out
 
 build-functional-test-image: ## Build Docker image needed to run functional tests
 	docker build -t="openfga/openfga:functionaltest" .
 
 .PHONY: functional-test
 functional-test: ## Run functional tests (needs build-functional-test-image)
-	go test $(gotest_extra_flags) -v \
-			-coverprofile=coveragefunctional.out \
-			-coverpkg=$(COVERPKG) \
-			-covermode=atomic -race \
+	go test $(gotest_extra_flags) -race -v \
```

Comment:
I realized the cover profile here is worthless (it reports less than 1%), and that's because the functional tests run against the docker image and so none of the source dependencies get covered in the coverage profile. So I'm removing them for that reason.r